src/ui/components/ui/buttons/Button.module.css

```diff
@@ -21,6 +21,10 @@
   border-radius: 0 4px 4px 0;
 }
 
+:global button.modal-close {
```

Comment:
Maybe move it here?
https://github.com/wavesplatform/waves-keeper/blob/KEEP-354/fix-button-paddings/src/ui/styles/app.styl#L443

Reply:
Forgot the button element has padding